package/MDAnalysis/exceptions.py

```diff
@@ -35,6 +35,24 @@ class SelectionError(Exception):
 class NoDataError(ValueError, AttributeError):
     """Raised when empty input is not allowed or required data are missing.
 
+    This exception is raised in the following scenarios:
+    * Raised when a :class:`~MDAnalysis.core.topologyattrs.TopologyAttr`
+    (e.g., bonds, charges) is not present in the data.
```

Comment:
```suggestion
    This exception is raised in the following scenarios:
    
    * Raised when a :class:`~MDAnalysis.core.topologyattrs.TopologyAttr`
      (e.g., bonds, charges) is not present in the data.
```

@HeetVekariya in multi-line items of a list, in order to render correctly I believe the subsequent lines need to be flush with the first, i.e. instead of:

```
    * item one
    second line of item one
```

It should be
```
    * item one
      second line of item one
```

You also typically need an empty line in between the normal text and the start of a list item. Unfortunately reStructuredText can be quite strict.

Reply:
- I think we should have visible bullet points, right ? If yes, can you show me how can i add it ?
![image](https://github.com/MDAnalysis/mdanalysis/assets/91054457/b896d7fd-f99e-451d-b1f0-7e5023294fd0)